packages/aws-serverless/src/sdk.ts

```diff
@@ -25,7 +25,7 @@ import { SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, SEMANTIC_ATTRIBUTE_SENTRY_SOURCE } fr
 import { DEBUG_BUILD } from './debug-build';
 import { awsIntegration } from './integration/aws';
 import { awsLambdaIntegration } from './integration/awslambda';
-import { markEventUnhandled } from './utils';
+import { getTraceData, markEventUnhandled } from './utils';
```

Comment:
m: Can we change the name here, as it conflicts with `getTraceData` we have in core, making this possibly a bit confusing 😅 E.g. `getAwsTraceData` or something like this?

Reply:
Yep, good point.

Reply:
Updated